webpack/stories/docs/APIMiddleware.md

```diff
@@ -46,3 +46,57 @@ dispatch({
   }
 });
 ```
+
+## Main API Reducer
+
+The API will manage the request lifecycle in the redux store > API > the provided key, e.g:
+```js
+{
+  API: {
+    MY_COMPONENT_KEY: { // this is the key you provided in the API action.
+      error: null,
+      status: 'RESOLVED',
+      results: [],
+    }
+  }
+}
+```
+it will manage the request statuses, errors and responses.
+
+### Formatters
+
+The are the default API responses formatters which you can look at in 'redux/API/APIHelpers.js',
+leaving the action without formatters will use the defaults.
+You can override these formatters and manipulate the response to update the state as you wish, e.g:
+```js
+dispatch({
+  type: API_OPERATIONS.GET,
+  key: COMPONENT_NAME,
+  url,
+  payload: data,
+  formatSuccess: ({ data, payload, status, error }) => ({}) // you can return whatever you like here.
+  formatFailure: ({ payload, status, error }) => ({}) // same here
+  formatPending: ({ payload, status, error }) => ({}) // same here
+});
+```
+
+A common use-case is to store each response per ID, you could achieve that by using the `idFormatters` from '/APIHelpers.js':
+```js
+dispatch({
+  type: API_OPERATIONS.GET,
+  key: COMPONENT_NAME,
+  url,
+  payload: data,
+  ...idFormatters, // this will do the default format for any response to be stored under [payload.id].
```

Comment:
Would like to see an example here of what `payload` would look like when using `idFormatters`.